hydra-node/src/Hydra/Network/Heartbeat.hs

```diff
@@ -89,46 +87,39 @@ withHeartbeat ::
   ) =>
   -- | This node's id, used to identify `Heartbeat` messages broadcast to peers.
   NodeId ->
-  -- | Callback listening to peers' status change as computed by the `withIncomingHeartbeat` layer.
-  ConnectionMessages m ->
   -- | Underlying `NetworkComponent` for sending and consuming `Heartbeat` messages.
   NetworkComponent m (Heartbeat inbound) (Heartbeat outbound) a ->
   -- | Returns a network component that can be used to send and consume arbitrary messages.
   -- This layer will take care of peeling out/wrapping messages into `Heartbeat`s.
   NetworkComponent m inbound outbound a
-withHeartbeat nodeId connectionMessages withNetwork =
-  withIncomingHeartbeat connectionMessages $
+withHeartbeat nodeId withNetwork =
```

Comment:
Must: How would the heartbeat work now? Where are `Connectivity` messages produced?